runtime/doc/api.txt

```diff
@@ -1350,6 +1350,27 @@ nvim_out_write({str})                                       *nvim_out_write()*
                 Parameters: ~
                     {str}  Message
 
+nvim_parse_cmdline({str}, {opts})                       *nvim_parse_cmdline()*
+                Parse command line.
+
+                Doesn't check the validity of command arguments.
+
+                Attributes: ~
+                    {fast}
```

Comment:
This is an argument in favor of keeping `nvim_parse_cmd` separate from `nvim_cmd` (which wouldn't be "fast" unless `opt.parse_only=true`).

Reply:
I think we should keep `nvim_parse_cmd` separate, but keep them compatible so that `nvim_cmd(nvim_parse_cmd(command))` always has the same effect as `nvim_command(command)`

Reply:
hmm. interesting...